test/common/schemaform/widgets/CurrencyWidget.unit.spec.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { expect } from 'chai';
+import SkinDeep from 'skin-deep';
+import sinon from 'sinon';
+
+import CurrencyWidget from '../../../../src/js/common/schemaform/widgets/CurrencyWidget';
+
+describe('Schemaform <CurrencyWidget>', () => {
+  it('should render', () => {
+    const tree = SkinDeep.shallowRender(
+      <CurrencyWidget
+          options={{}}
+          value={178}/>
+    );
+    expect(tree.subTree('input').props.value).to.equal('178.00');
```

Comment:
Minor efficiency boost: Pull out `const input = tree.subTree('input');` and use that in each of these `expect`s.